dev/diffs/4.0.0.diff

```diff
@@ -2211,20 +2578,19 @@ index 4fb8faa43a3..984fd1a9892 100644
        checkAnswer(
          // "fruit" column in this file is encoded using DELTA_LENGTH_BYTE_ARRAY.
 diff --git a/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetQuerySuite.scala b/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetQuerySuite.scala
-index a329d3fdc3c..ebca1264eb1 100644
+index bba71f1c48d..a45d3f6e797 100644
 --- a/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetQuerySuite.scala
 +++ b/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetQuerySuite.scala
-@@ -971,7 +971,8 @@ abstract class ParquetQuerySuite extends QueryTest with ParquetTest with SharedS
-     }
-   }
+@@ -48,6 +48,8 @@ abstract class ParquetQuerySuite extends QueryTest with ParquetTest with SharedS
+   import testImplicits._
  
--  test("SPARK-26677: negated null-safe equality comparison should not filter matched row groups") {
-+  test("SPARK-26677: negated null-safe equality comparison should not filter matched row groups",
-+    IgnoreCometNativeScan("Native scans had the filter pushed into DF operator, cannot strip")) {
-     withAllParquetReaders {
-       withTempPath { path =>
-         // Repeated values for dictionary encoding.
-@@ -1042,7 +1043,8 @@ abstract class ParquetQuerySuite extends QueryTest with ParquetTest with SharedS
+   test("simple select queries") {
++    println("==== Runtime Classpath ====")
++    System.getProperty("java.class.path").split(":").foreach(println)
```

Comment:
nit: looks like debug logging that could be removed?

Reply:
Removed. Thanks!